modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -225,6 +225,30 @@ This ID can be chosen from the config paramater section `wnd.kerberos.servers` a
 This is the keytab file that has been described in xref:enterprise/authentication/kerberos.adoc#keytab-files[Keytab Files].
 * `ocservice` +
 This is the SPN of the service user that has been described in xref:enterprise/authentication/kerberos.adoc#service-principal-name-spn[Service Principal Name (SPN)].
+* `usermapping` +
+Define a usermapping if necessary or required.
+
+{empty}
+
+The usermapping key::
++
+Note that `usermapping` is only possible with the enabled and configured {oc-marketplace-url}/apps/user_ldap[LDAP Integration] app. WND will throw an error if not.
```

Comment:
This isn't exactly true.
We have 3 different user mappings: "noop", "removeDomain" and "EALdapAttr". Only the last one requires LDAP in order to work properly. The rest don't have any requirement.

Reply:
fixed